compiler/luci/service/src/ShapeInfer_StridedSlice.cpp

```diff
@@ -45,21 +45,23 @@ const loco::DataType S32 = loco::DataType::S32;
 
 using int8 = int8_t;
 using int16 = int16_t;
+using int32 = int32_t;
+using uint32 = uint32_t;
 
 struct StridedSliceParams
 {
-  int8 start_indices_count = 0;
-  int16 start_indices[kMaxDim];
-  int8 stop_indices_count = 0;
-  int16 stop_indices[kMaxDim];
-  int8 strides_count = 0;
-  int16 strides[kMaxDim];
-
-  int16 begin_mask = 0;
-  int16 ellipsis_mask = 0;
-  int16 end_mask = 0;
-  int16 new_axis_mask = 0;
-  int16 shrink_axis_mask = 0;
+  uint32 start_indices_count = 0;
+  int32 start_indices[kMaxDim];
+  uint32 stop_indices_count = 0;
+  int32 stop_indices[kMaxDim];
+  uint32 strides_count = 0;
+  int32 strides[kMaxDim];
+
+  int32 begin_mask = 0;
+  int32 ellipsis_mask = 0;
+  int32 end_mask = 0;
+  int32 new_axis_mask = 0;
+  int32 shrink_axis_mask = 0;
```

Comment:
these are from reference code. why do you want to change thee?

Reply:
@seanshpark 
because  in e.g. https://github.com/Samsung/ONE/blob/a13d56cffc2704c703b059db4c136f7d149fd2a7/compiler/luci/service/src/ShapeInfer_StridedSlice.cpp#L331 there is an assignment of 32-bit integer to 16-bit integer, so if an integer will be greater than 2^(16)/2=32768, there will be a flow issue.

Reply:
@seanshpark
Please see relevant part of tensorflow:
https://github.com/tensorflow/tensorflow/blob/388d952114e59a1aeda440ed4737b29f8b7c6e8a/tensorflow/lite/kernels/internal/types.h#LL972C1-L985C3.

Reply:
Could you sync the types with TF? e.g., counts are int8 and masks are uint16 in TF. It would be better to use the same type and insert overflow checks in proper places. Our basic principle is to sync with TF for TF-existing Ops.

Reply:
> Could you sync the types with TF?

Ahh. Ok. I see. Thank you.

Reply:
I would suggest not to change existing codes from TF.
Current reference code is from TF 2.6.0 (or maybe TF 2.8.0)
If you must want to change them for any problems with any models, then refer TF 2.10.x codes.

Reply:
For 2.10.1 tensorflow `*indices` and `strides` of `StridedSliceParams` struct are changed to 32bits. Please see `types.h` attached.
[types.h.txt](https://github.com/Samsung/ONE/files/11550817/types.h.txt)

Reply:
done

Reply:
please see
https://github.com/tensorflow/tensorflow/commit/79605069321520bd8af318eef92b71070dcc8961
commit fixing `indices` to 32 bits.